recipes/tcsh/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -eu -o pipefail
+
+./configure --prefix="${PREFIX}"
```

Comment:
Please add ` || { cat config.log; exit 1; }` on this line. Should give us some more details about the cause of the failure.

cc @isuruf